pkg/venafi/firefly/connector.go

```diff
@@ -228,7 +232,7 @@ func (c *Connector) SynchronousRequestCertificate(req *certificate.Request) (cer
 	return certificates, nil
 }
 
-func (c *Connector) getCertificateRequest(req *certificate.Request) *certificateRequest {
+func (c *Connector) getCertificateRequest(req *certificate.Request) (*certificateRequest, error) {
```

Comment:
at what point an error is returned by this function? No need to return an error if you never throw one

Reply:
The error commented in the below comment it's the one can be thrown

Reply:
Ok, I totally lost grasp of the scope